### PR TITLE
ACS-8440 Retry endpoint.

### PR DESCRIPTION
**Pull request by tpage-alfresco**
Also fix for feedback endpoint to add default constructor.

### CODE REVIEW

File: hxinsight-extension/src/main/java/org/alfresco/hxi_connector/hxi_extension/rest/api/model/QuestionModel.java [hunk 1]
```diff
@@ -43,6 +44,7 @@
 @Getter
 @AllArgsConstructor
 @NoArgsConstructor
+@EqualsAndHashCode
```
Comment (krdabrowski) [inline hxinsight-extension/src/main/java/org/alfresco/hxi_connector/hxi_extension/rest/api/model/QuestionModel.java:47]: Maybe `@Data` will be more suitable here? 🤔

File: hxinsight-extension/src/main/java/org/alfresco/hxi_connector/hxi_extension/service/model/RestrictionQuery.java [hunk 2]
```diff
@@ -30,11 +30,13 @@
 
 import lombok.AllArgsConstructor;
 import lombok.Data;
+import lombok.EqualsAndHashCode;
 import lombok.NoArgsConstructor;
 
 @Data
 @AllArgsConstructor
 @NoArgsConstructor
+@EqualsAndHashCode
```
Comment (krdabrowski) [inline hxinsight-extension/src/main/java/org/alfresco/hxi_connector/hxi_extension/service/model/RestrictionQuery.java:39]: This should be already included within `@Data`

**Review by krdabrowski — APPROVED** [review]
Have small remarks, but overall LGTM

**Review by azakrzewski-hy — APPROVED** [review]
LGTM

**Pull request merged**